api.php

```diff
@@ -980,15 +980,37 @@ public function run($userID, $permissionIsExplicit) {
 		if ($args['message'] === null)
 			throw new RequestException('message is required.');
```

Comment:
We should return json as well for these as well. if ($args['toCountryID'] === null) if ($args['message'] === null). Since were modifying an existing API was this being used anywhere? We want to make sure we don't break anything currently being used.